src/platform/unix.rs

```diff
@@ -42,6 +48,28 @@ impl WindowExtUnix for Window {
   fn set_skip_taskbar(&self, skip: bool) {
     self.window.set_skip_taskbar(skip);
   }
+
+  fn set_progress_bar(&self, progress: ProgressBarState) {
+    if let Some(state) = progress.state {
+      let taskbar_state = {
+        match state {
+          TaskbarProgressState::None => false,
+          _ => true,
+        }
+      };
+
+      self
+        .window
+        .set_taskbar_progress_state(taskbar_state, progress.unity_uri.clone());
+    }
+    if let Some(value) = progress.progress {
+      let progress_num = if value > 100 { 100f64 } else { value };
+
+      self
+        .window
+        .set_taskbar_progress(progress_num, progress.unity_uri.clone());
+    }
+  }
```

Comment:
remove this logic, create a method `set_progress_bar` in `src/platform_impl/linux/window.rs` and move this logic there

Reply:
✅